src/Legacy/Numbers/Words/Locale/Yo.php

```diff
@@ -0,0 +1,238 @@
+		9 => 'mẹsan'
+	];
+
+	protected static $teens = [
+		10 => 'mẹwa',
+		11 => 'mọkanla',
+		12 => 'mejila',
+		13 => 'mẹtala',
+		14 => 'mẹrinla',
+		15 => 'mẹẹdogun',
+		16 => 'mẹrindilogun',
+		17 => 'mẹtala',
+		18 => 'mejidilogun',
+		19 => 'mọkandinlogun',
+	];
+
+	protected static $tens = [
+		20 => 'ogun',
+		30 => 'ọgbọn',
+		40 => 'ogoji',
+		50 => 'aadọta',
+		60 => 'Ogota',
+		70 => 'aadọrin',
+		80 => 'ọgọrin',
+		90 => 'aadọrun',
+	];
+
+	protected static $hundred = [
+		0 => '',
+		100 => 'ọgọrun',
+		200 => 'igba',
+		300 => 'ọ̀ọ́dúrún',
+		400 => 'irinwó',
+		500 => 'ẹdẹgbẹta',
+		600 => 'ẹgbẹta',
+		700 => 'ẹẹdẹgbẹrin',
+		800 => 'ẹgbẹ̀rin',
+		900 => 'ẹ̀ẹ́dẹ́gbẹ̀rún',
+	];
+
+	protected static $mega = [
+		1 => 'ẹgbẹrun',
+		2 => 'miliọnu',
+		3 => 'biliọnu',
+		4 => 'miliọnu miliọnu',
+		5 => 'biliọnu miliọnu',
+		6 => 'biliọnu biliọnu',
+		7 => 'aimoye miliọnu',
+	];
+
+	public static $currencyNames = [
+		'ALL' => [['lek'], ['qindarka']],
+		'AUD' => [['Australian dollar'], ['cent']],
+		'BAM' => [['convertible marka'], ['fenig']],
+		'BGN' => [['lev'], ['stotinka']],
+		'BRL' => [['real'], ['centavos']],
+		'BYR' => [['Belarussian rouble'], ['kopiejka']],
+		'CAD' => [['Canadian dollar'], ['cent']],
+		'CHF' => [['Swiss franc'], ['rapp']],
+		'CYP' => [['Cypriot pound'], ['cent']],
+		'CZK' => [['Czech koruna'], ['halerz']],
+		'DKK' => [['Danish krone'], ['ore']],
+		'DZD' => [['dinar'], ['cent']],
+		'EEK' => [['kroon'], ['senti']],
+		'EUR' => [['euro'], ['euro-cent']],
+		'GBP' => [['pound', 'pounds'], ['pence', 'pence']],
+		'HKD' => [['Hong Kong dollar'], ['cent']],
+		'HRK' => [['Croatian kuna'], ['lipa']],
+		'HUF' => [['forint'], ['filler']],
+		'ILS' => [['new sheqel', 'new sheqels'], ['agora', 'agorot']],
+		'ISK' => [['Icelandic króna'], ['aurar']],
+		'JPY' => [['yen'], ['sen']],
+		'LTL' => [['litas'], ['cent']],
+		'LVL' => [['lat'], ['sentim']],
+		'LYD' => [['dinar'], ['cent']],
+		'MAD' => [['dirham'], ['cent']],
+		'MKD' => [['Macedonian dinar'], ['deni']],
+		'MRO' => [['ouguiya'], ['khoums']],
+		'MTL' => [['Maltese lira'], ['centym']],
+		'NGN' => [['Naira'], ['kobo']],
+		'NOK' => [['Norwegian krone'], ['oere']],
+		'PHP' => [['peso'], ['centavo']],
+		'PLN' => [['zloty', 'zlotys'], ['grosz']],
+		'ROL' => [['Romanian leu'], ['bani']],
+		'RUB' => [['Russian Federation rouble'], ['kopiejka']],
+		'SEK' => [['Swedish krona'], ['oere']],
+		'SIT' => [['Tolar'], ['stotinia']],
+		'SKK' => [['Slovak koruna'], []],
+		'TMT' => [['manat'], ['tenge']],
+		'TND' => [['dinar'], ['cent']],
+		'TRL' => [['lira'], ['kuruş']],
+		'UAH' => [['hryvna'], ['cent']],
+		'USD' => [['dollar'], ['cent']],
+		'XAF' => [['CFA franc'], ['cent']],
+		'XOF' => [['CFA franc'], ['cent']],
+		'XPF' => [['CFP franc'], ['centime']],
+		'YUM' => [['dinar'], ['para']],
+		'ZAR' => [['rand'], ['cent']],
+	];
+
+    /**
+     * @param $num
+     * @return string
+     */
+	protected function toWords($num)
+	{
+		$word = false;
```

Comment:
Perhaps `$word` should be an empty string? Later on there's a concatenation on this variable.